### PR TITLE
[DO NOT MERGE] Output tool features

### PR DESCRIPTION
**Pull request by johnmu**
this is just to ask @marghoob questions :)

### CODE REVIEW

File: metasv.py [hunk 1]
```diff
@@ -169,14 +189,21 @@
...
+  # Do the intra-tool merging
   for tool in tools:
     if sv_type not in intervals[tool]: continue
     logger.info("First level merging for %s for tool %s" % (sv_type, tool))
     tool_merged_intervals[sv_type] += sv_interval.merge_intervals(intervals[tool][sv_type])
+
+  # Do the inter-tool merging
   merged_intervals = sv_interval.merge_intervals(tool_merged_intervals[sv_type])
+
+  # Marghoob.... what does this do? :/
```
Comment (johnmu) [inline metasv.py]: @marghoob Can you comment on this?

Comment (marghoob) [inline metasv.py]: This merges all the calls. Later on, this merged set is used to generate a merged set where the contributing intervals are a significant fraction of the merged interval.

File: sv_interval.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def __init__(self, chrom=None, start=0, end=0, name=None, sv_type=None, length=0
     self.is_precise = False
     self.is_validated = False
     self.validating_interval = None
-    self.info_dict = {}
+    self.info_dict = {} # Marghoob, did you have any specific plans for this?
```
Comment (johnmu) [inline sv_interval.py]: @marghoob did you have have specific plans for this as opposed to "info"? Was it just supposed to mirror info?

Comment (marghoob) [inline sv_interval.py]: It was supposed to mirror info in the input VCFs. I was planning on retaining the original information. Now that we are keeping the original stuff in native_sv field, this doesn't seem necessary.

**Comment by johnmu** [comment]
@marghoob feel free to merge this whenever you dare

**Pull request merged**